rviz_rendering/src/rviz_rendering/render_system.cpp

```diff
@@ -239,6 +239,16 @@ RenderSystem::detectGlVersion()
       }
       break;
   }
+
+  #ifdef __linux__
+  // TODO(pijaro): Nouveanu and Intel drivers don't support glsl150 even if they "should".
+  // We fix version to 120 since this is the one we are currently using in materials.
```

Comment:
@pijaro Sorry, this PR dropped off my radar. 

It sounds like it would be good to merge this workaround until we resolve the core issue.
Could you reference the open ticket here in this comment? https://github.com/ros2/rviz/issues/691